compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeWindowPanel.desktop.kt

```diff
@@ -101,6 +101,11 @@ internal class ComposeWindowPanel(
             override fun getDefaultComponent(aContainer: Container?) = null
         }
         isFocusCycleRoot = true
+
+        if (ComposeFeatureFlags.useSwingGraphics) {
+            println("Usage of \"compose.swing.render.on.graphics\" affect only behaviour of ComposePanel." +
+                "Usage together with ComposeWindow or ComposeDialog won't have any effect.")
+        }
```

Comment:
I'm not sure about printing this. The flag is for the entire process, but you can have both `ComposePanel` and `ComposeWindow` in a single app...

Reply:
True. The intention here is to notify already wrong configured applications. Ideas are welcome

Reply:
Already thought about it, so yes we can. But I don't really won't to remove global flag because despite "experimental" status it's widely adopted

Reply:
> Why is this a global flag, though?

It was introduced in #601, before "mediator" refactor. So I guess it was easier at the time

Reply:
Exposed as another optional/experimental parameter in `ComposePanel`'s constructor

Reply:
Great.
As for this print, I think I personally would prefer not to pollute the output, but I'd leave the decision to @igordmn .
If we do add it, probably better to print to `System.err`, though (we're in `desktopMain` here).